cmd/config/set/set.go

```diff
@@ -33,10 +36,22 @@ func NewConfigSetCommand() *cobra.Command {
 	return &cobra.Command{
 		Use:   "set [flags] VARIABLE VALUE",
 		Short: "Sets the variable's value",
-		Long:  "Sets the value of a config variable. See 'rosa config --help' for supported config variables.",
-		Args:  cobra.ExactArgs(2),
-		Run:   run,
+		Long: fmt.Sprintf("Sets the value of a config variable. Supported variables are: %s.",
+			strings.Join(getAllowedConfigProperties(), ", ")),
+		Args: cobra.ExactArgs(2),
+		Run:  run,
+	}
+}
+
+func getAllowedConfigProperties() []string {
```

Comment:
@tirthct Same comment here about moving this into `config.go`. And also a unit test.

Reply:
Moved.